lightning/src/events/mod.rs

```diff
@@ -1437,6 +1448,17 @@ impl MaybeReadable for Event {
 				};
 				f()
 			},
+			33u8 => {
```

Comment:
I guess that would mean a user couldn't downgrade if they have this event serialized. Do I have that right? I don't know the right answer here. If the user *needs* to downgrade then this could prevent them from doing so.